src/App.tsx

```diff
@@ -15,37 +16,104 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+type Good = string;
+
+enum SortType {
+  NONE = '',
+  ALPHABET = 'alphabet',
+  LENGTH = 'length',
+}
+
+const GoodList: React.FC<{ goods: Good[] }> = ({ goods }) => (
```

Comment:
It's a bad practice to create two components in the one file.
You don't need a separate component for this logic, or if you want to create one, you need to export it to a separate file